src/RestEase/Implementation/Requester.cs

```diff
@@ -272,20 +272,30 @@ string Encode(string data)
         /// <returns>null if no body is set, otherwise a suitable HttpContent (StringContent, StreamContent, FormUrlEncodedContent, etc)</returns>
         protected virtual HttpContent ConstructContent(IRequestInfo requestInfo)
         {
-            if (requestInfo.BodyParameterInfo == null || requestInfo.BodyParameterInfo.ObjectValue == null)
```

Comment:
My plan was just to check if the BodyParameterInfo was no null, but the ObjectValue was null, and construct an empty body if so. That means that someone supplied a [Body] but passed null. No other changes required.

Reply:
Ok then, will it be acceptable if i will change first **if** statement to this

`            if (requestInfo.BodyParameterInfo == null )
                return null;
`
and then add new statement

`            if (requestInfo.BodyParameterInfo != null && requestInfo.BodyParameterInfo.ObjectValue == null)
                return new ByteArrayContent(new byte[0]);
`
?